Box-list fixes

MariCerv opened the pull request:
- new button fixed
- container nabber fixed
- background color ( boxes and page ) fixed
- font size box-extra-info fixed
- modal button “create” fixed

MariCerv committed:
Box-list fixes

- new button fixed
- container nabber fixed
- background color ( boxes and page ) fixed
- font size box-extra-info fixed
- modal button “create” fixed

MariCerv committed:
button create box fixed
highlight created box

pjaneiro commented:
I've checked with @MariCerv and it's working, but she still wants to change the way new boxes are highlighted, don't merge it yet

MariCerv committed:
Font modals fixed + testing datepicker

MariCerv committed:
highlight created box finished

MariCerv committed:
minor fix modal + Datepicker fixed

MariCerv committed:
Modal fixes

-padding button create fixed (modal)
- padding + color inputs, textarea fixed (modal)

dethos commented on boxes/templates/boxes/box_list.html:
```diff
@@ -62,120 +62,127 @@ <h3 class="box__title text text-darkest">{{box.name}}</h3>
-              {% endif %}
+              <p class="no-margin-bottom">
+                <span class="smalltext label-blue">Messages:</span>
+                <span class="smalltext text-darkest">{{box.messages.count}}/{{box.max_messages}}</span></p>
```
The indentation does not seem to be correct here, and bellow.

dethos commented on boxes/templates/boxes/box_list.html:
```diff
@@ -62,120 +62,127 @@ <h3 class="box__title text text-darkest">{{box.name}}</h3>
+    <script src="{% static 'javascripts/box_list.js' %}"></script>
+    <script type="text/javascript" src="{% static 'javascripts/jquery.datetimepicker.min.js' %}"></script>
+    <script src="{% static 'javascripts/box_create.js' %}"></script>
+    <script type="text/javascript">
```
We are trying to avoid inline javascript. Please move the code to `javascripts/box_create.js` or add this classes to the elements, since this seems to run every time.

pjaneiro replied:
It's not easy to add the classes to the elements since they're created dynamically by Django...

dethos replied:
@Budgerous ok, so moving to the external file seems to be the right solution.

dethos reviewed (changes requested):
Overall it seems good, but the indentation is not correct due to a subtle issue on one file and we can't allow inline JS. As soon as it is fixed I will merge.

MariCerv committed:
indentation + js fixed
